src/mono/wasm/runtime/modularize-dotnet.pre.js

```diff
@@ -0,0 +1,11 @@
+const MONO = {}, BINDING = {}, INTERNAL = {};
+var ENVIRONMENT_IS_GLOBAL = typeof globalThis.Module === "object";
+if (ENVIRONMENT_IS_GLOBAL) {
+    globalThis.Module.ready = Module.ready;
```

Comment:
Let's check to see whether a 'ready' already exists on Module here and if so... throw or something, since we've previously had people load us alongside another global-scoped emscripten module and then file confused issues about it.
Doing that is incorrect and they shouldn't expect it to work, we might as well fail early so they know what they did wrong.

Reply:
We may actually want to check for a non-empty Module period and reject that, but I assume in some cases they're pre-creating it and putting stuff in there. We might want to check for some specific property emscripten puts there once it's loaded something

Reply:
In both modularized and global `Module` it will be merged into implementation `Module`. Emscripten is mostly designed to let user shoot himself in to foot with any "override". I think we should stay consistent.

Rejecting non-empty Module/EmscriptenModuleConfig doesn't work, because that's the only startup API we and Emscripten give. 

Specifically `Module.ready` is always replaced by emscripten or us. I added the check.